bedrock/privacy/templates/privacy/privacy-day.html

```diff
@@ -1,142 +1,310 @@
+{% endblock %}
 
-{% block extrahead %}
+{% block body_id %}privacy-day{% endblock %}
+
+{% block site_css %}
   {{ css('privacy-day') }}
 {% endblock %}
 
-{% block body_class %}privacy-day{% endblock %}
+{%
+set share_urls = {
+    'twitter': 'http://mzl.la/1E63Mxm',
+    'googleplus': 'https://www.mozilla.org/' + LANG + '/privacy/tips/?utm_source=Google&utm_medium=MozLandingPage&utm_campaign=privacy2015&utm_content=WebsiteShare',
+    'facebook': 'https://www.mozilla.org/' + LANG + '/privacy/tips/?utm_source=Facebook&utm_medium=MozLandingPage&utm_campaign=privacy2015&utm_content=WebsiteShare#play'
+}
+%}
 
 {% block site_header %}
   <header id="masthead">
     <div class="container">
       <a href="{{ url('mozorg.home') }}" id="tabzilla" data-infobar="{{ settings.TABZILLA_INFOBAR_OPTIONS }}">Mozilla</a>
-    {% block site_header_logo %}
-      <h2><a href="{{ url('mozorg.home') }}">{{ high_res_img('img/sandstone/mozilla-wordmark-white.png', {'alt': 'Mozilla', 'width': '170', 'height': '46'}) }}</a></h2>
-    {% endblock %}
+      {% block site_header_nav %}{% endblock %}
     </div>
   </header>
 {% endblock %}
 
 {% block content %}
+<main role="main" id="main-content">
+  <div id="page-header">
+    <div class="container">
+      <h1>{{ _('Get smart on privacy') }}</h1>
+      <h2>{{ _('In four easy steps') }}</h2>
+    </div>
+  </div>
 
-<main role="main">
-  <article id="main-content">
-    <header>
-      <div class="container">
-        <h1>{{ _('We put your privacy first') }}</h1>
-        <div id="download">
-          {{ download_firefox(small=True) }}
-        </div>
-        <p id="principles"><a href="{{ url('privacy.principles') }}">{{ _('Explore our privacy principles') }}</a></p>
-      </div>
-    </header>
+  <div class="pager pager-auto-init pager-with-tabs">
 
+  <div id="button-nav-wrapper" class="button-wrapper">
     <div class="container">
-      <div class="intro">
-        <p>
-        {% trans manifesto=url('mozorg.about.manifesto') %}
-          At Mozilla, we have a long history of taking privacy seriously. It’s front and center
-          in every piece of software we make. It's in our <a href="{{ manifesto }}">Manifesto</a>.
-        {% endtrans %}
-        {% trans %}
-          It’s part of our DNA. The Web is an amazing thing and it's at its best when it’s open,
-          transparent and safe for everyone. Here are some of the ways we work to give you a better
-          understanding — and more control — of your privacy online.
-        {% endtrans %}
-        </p>
+      <div class="share-widget">
+        {{ share_cta(_('Share'), share_urls, _('I’m taking control of my online privacy. Here’s how you can, too: #SmartOnPrivacy')) }}
       </div>
+      <nav id="tips-nav-direct">
+        <ol role="tablist" class="pager-tabs">
+          <li role="presentation">
+            <a role="tab" aria-controls="ask" aria-selected="true" href="#ask" id="tab-ask"><span>{{ _('1') }}</span> {{ _('ASK') }}</a>
```

Comment:
I wondered that as well, but we have run into cases in the past where the text-transform is problematic in some locales. I thought this would give that freedom to the localizer, but perhaps it would be best to just black-list some locales from the text-transform CSS.

Probably too late for these strings for now though. I would like to get a clear decision on this for future work though. CC: @flodolo @pascalchevrel